src/App.js

```diff
@@ -1,5 +1,93 @@
+const { Random } = require('@woowacourse/mission-utils');
+const CONSTANT = require('./constant/constants.js');
+const Lotto = require('./Lotto.js');
+const Log = require('./views/Log.js');
+
+const log = new Log();
+
 class App {
-  play() {}
+  #lottos;
+
+  #winner;
+
+  #amount;
+
+  #prize;
+
+  constructor() {
+    this.#lottos = [];
+    this.#winner = [];
+    this.#amount = 0;
+    this.#prize = 0;
+  }
+
+  start(answer) {
+    this.validateAmount(answer);
+    this.#amount = +answer;
+    this.generateLotto(this.#amount / CONSTANT.UNIT);
+    log.printLottos(this.#amount, this.#lottos);
+    log.getWinner(this.resultAndBonusHandler.bind(this));
+  }
+
+  validateAmount(amount) {
+    const { CHECK, ERROR_MSG } = CONSTANT;
+    if (!CHECK.IS_NUMBER(amount)) throw new Error(`${ERROR_MSG.NAN}`);
+    if (CHECK.IS_UNIT(amount)) throw new Error(`${ERROR_MSG.WRONG_AMOUNT}`);
+  }
+
+  generateLotto(num) {
+    this.#lottos = Array.from(
+      { length: num },
+      () => new Lotto(Random.pickUniqueNumbersInRange(1, 45, 6)),
+    );
+  }
+
+  resultAndBonusHandler(answer) {
+    this.compareResults(answer);
+    log.getBonus(this.end.bind(this));
+  }
+
+  compareResults(numbers) {
+    this.#winner = new Lotto(numbers.split(',').map(num => +num)).numbers;
```

Comment:
```suggestion
    this.#winner = new Lotto(numbers.split(',').map(Number)).numbers;
```
값을 숫자로 바꿀 때 + 연산자를 사용하는 것도 좋지만, 가독성을 위해 Number를 사용하면 더 좋겠습니다 😄